### ISSUE TITLE
bounce processing, auto-correct misspelled usernames based on name

### ISSUE THREAD

**Issue by rrustema** [petities/petitions.eu#337]
There are patterns in the usernames of signatories. Often they use parts of their real name. But the typos also occur mostly in the username part. 

In a future release we could process bounces by comparing the username with their name. 
• After two unconfirmed reminders we compare the username part of the e-mail address with the name. 
• If an entire name returns in this part, except one character, 
• and this character is nearby on a QWERTY-keyboard, it is likely to be a typo. 
• Then we compare with the database if this e-mail address is known to us as a signatory
• and if the confirmed name is similar to the unconfirmed name,
• And the user did not already correct the mistake and sign correctly (two weeks have passed after all)

Then, if these six conditions are met, we send a special e-mail that we think the person tried to sign but made a typo. The link to confirm included.

The question, is this feasible enough to file as a 'nice to have' feature?

**Comment by skoolstra** [comment]
Removing 'nice to have', adding 'enhancement'. We should think about this idea a bit more before we start building it.
